.github/workflows/rh-support.yml

```diff
@@ -4,6 +4,9 @@ on:
     branches:
       - master
 
+permissions:
```

Comment:
This change LGTM - agreed @alexrashed ?

Reply:
I agree, looking good! Thanks for the contribution, @naveensrinivasan!